E2a-TRAS/GL01/Dataset/README.md

```diff
@@ -0,0 +1,24 @@
+# Dataset description
+
+| Documentation Code  | Test Title                           | Exchange Code | Test Code | Author          | Data Owner | Version | Date       |
+|---------------------|--------------------------------------|---------------|-----------|-----------------|------------|---------|------------|
+| IFC4.3AbRV_E2a_GL01 | (RFI) Global Positioning RFI dataset | E2a (TRAS)    | GL01      | Evandro Alfieri | RFI        | 1.0     | DD.MM.YYYY |
+
+## Model Dataset
+
+This test case utilises the following dataset.
+
+| Filename                               | Type (format) | Description                                               |
+|----------------------------------------|---------------|-----------------------------------------------------------|
+| [GeographicData](./GeographicData.png) | table (png)   | [Geographic data relevant for the test](#Geographic-Data) |
+
+
+### Geographic Data
+
+The table below captures the properties or the Geographic Coordinate System (GCS) and Projected Coordinate System (PGC), usefull for the test case mentioned in the header.
```

Comment:
Please use *Coordinate Reference System* here, not only *Coordinate System*.

Reply:
Done